orcid-core/src/main/java/org/orcid/core/manager/impl/OrcidJaxbCopyManagerImpl.java

```diff
@@ -342,9 +342,14 @@ private void copyUpdatedEmails(ContactDetails existingContactDetails, ContactDet
 				}
 			}
 		}
+		//Set primary = false for each remaining new email.
 		if(updatedContactDetails != null) {
-			allEmails.addAll(updatedContactDetails.getEmail());
+			for(Email newEmail : updatedContactDetails.getEmail()) {
```

Comment:
Wanted to make sure we tested a client creating a record. http://members.orcid.org/api/tutorial-create-new-record-using-curl-12-and-earlier